cmd/brew-bootstrap-rbenv-ruby

```diff
@@ -80,6 +80,9 @@ if ! rbenv version-name &>/dev/null; then
     fi
   fi
 
+  if [ "$(uname -sm)" = "Darwin arm64" ]; then
+    export RUBY_CFLAGS=-DUSE_FFI_CLOSURE_ALLOC
+  fi
```

Comment:
```suggestion
  # use workaround for necessary for some Ruby versions
  # https://github.com/ffi/ffi/issues/869#issuecomment-752123090
  if [ "$(uname -sm)" = "Darwin arm64" ]; then
    export RUBY_CFLAGS=-DUSE_FFI_CLOSURE_ALLOC
  fi
```

Reply:
Oops! Great suggestion, thanks for the feedback! ❤️